sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -572,6 +572,7 @@ class ExplainSuiteAE extends ExplainSuiteHelper with EnableAdaptiveExecutionSuit
       """
         |(11) AQEShuffleRead
         |Input [5]: [k#x, count#xL, sum#xL, sum#x, count#xL]
+        |Arguments: coalesced
```

Comment:
just for curiosity: which code change leads to this?

Reply:
There isn't one in this PR.  This line was part of the actual output before. The test checks if the actual plan contains
```
      """
        |(11) AQEShuffleRead
        |Input [5]: [k#x, count#xL, sum#xL, sum#x, count#xL]
        |Arguments: coalesced
        |""".stripMargin,
```
which is true in both versions.